aiogram/utils/helper.py

```diff
@@ -233,3 +235,60 @@ def all(cls) -> List[str]:
             else:
                 result.append(value)
         return result
+
+
+class DefaultProperty(Generic[T]):
+    """
+    Implements descriptor. Intended to be used as a class attribute and only internally.
+    """
+
+    __slots__ = (
+        "name_resolver",
+        "name",
+        "fget",
+    )
+
+    def __init__(
+        self,
+        default: Optional[T] = None,
+        *,
+        name_resolver: Callable[[str], str] = lambda s: "_" + s,
+        fget: Optional[Callable[[Any], T]] = None,
```

Comment:
`fget` means something like "factory get"?
So if answer for my question - "Yes", this argument should be renamed to `factory`

Reply:
it's actually came from python's built-in `property` parameter `fget`. So no, it's a function to be used for getting an attribute value

Reply:
ok, you can leave it as it is